controllers/tradeHistory.js

```diff
@@ -18,7 +18,7 @@ const options = {
     headers: {
     "Content-Type": "application/json",
     "x-api-key":
-        process.env.x_api_key,
+    process.env.X_API_KEY,
```

Comment:
I already did this in my pull request that was merged

Reply:
I know that.